json_templates/azure/c0-azure-base/ert-template.json

```diff
@@ -65,46 +65,325 @@
         "value": "mysql-proxy-lb.sys.{{pcf_ert_domain}}"
       },
       ".mysql_monitor.recipient_email": {
-        "value" : "c0-coreteam@pivotal.io"
+        "value": "c0-coreteam@pivotal.io"
```

Comment:
@krishicks This is a pre-existing code we can create a new bug or story in the tracker to get to this. The scope of this story was just to support the azure blob store for ERT tile.